### PR TITLE
Create homepage network selector component

### PR DESCRIPTION
**Pull request by buberdds**
Network switcher used in home page. PR covers only UI. Logic for switching links in graph will be covered in next PR.

### CODE REVIEW

File: src/app/pages/HomePage/Graph/NetoworkSelector/index.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        )}
+        <Box sx={{ height: 30, display: 'flex' }}>
+          {options.map(option => (
+            <Collapse orientation="horizontal" in={open || network === option} key={option}>
+              <StyledButton
+                onClick={() => setNetwork(option as Network)}
+                size="small"
+                variant="outlined"
+                sx={{ ml: 4 }}
+                selectedNetwork={option === network}
```
Comment (lukaw3d) [inline src/app/pages/HomePage/Graph/NetoworkSelector/index.tsx:86]: I would rename to isSelectedNetwork

File: src/app/pages/HomePage/Graph/NetoworkSelector/index.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        {!isMobile && (
+          <Typography component="span" sx={{ fontSize: '12px', color: COLORS.white }}>
+            {t('home.selectNetwork')}
+          </Typography>
+        )}
+        <Box sx={{ height: 30, display: 'flex' }}>
+          {options.map(option => (
+            <Collapse orientation="horizontal" in={open || network === option} key={option}>
+              <StyledButton
+                onClick={() => setNetwork(option as Network)}
```
Comment (lukaw3d) [inline src/app/pages/HomePage/Graph/NetoworkSelector/index.tsx:82]: ToggleButton seems like a nicer component to use here

File: src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx [hunk 3]
```diff
@@ -16,7 +16,9 @@ import { GraphUtils } from '../Graph/graph-utils'
 import useResizeObserver from 'use-resize-observer'
 import HelpScreen from '../HelpScreen'
 import { GraphTooltipMobile } from '../GraphTooltip/GraphTooltipMobile'
+import { NetworkSelector } from '../NetoworkSelector'
```
Comment (lukaw3d) [inline src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx:19]: `'../NetoworkSelector'` typo

File: src/app/pages/HomePage/Graph/NetoworkSelector/index.tsx [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+                size="small"
+                variant="outlined"
+                sx={{ ml: 4 }}
+                selectedNetwork={option === network}
+              >
+                {labels[option as Network]}
```
Comment (lukaw3d) [inline src/app/pages/HomePage/Graph/NetoworkSelector/index.tsx:88]: Instead of `option as Network`: `const options: Network[] = ['mainnet', 'testnet']`

**Review by lukaw3d — APPROVED** [review]

**Comment by csillag** [comment]
> Network switcher used in home page. PR covers only UI. Logic for switching links in graph will be covered in next PR.

The logic is now handled by https://github.com/oasisprotocol/explorer/pull/402

**Pull request merged**